### PR TITLE
postgresql: build btree_gin

### PR DESCRIPTION
**Pull request by aicynide**
Build btree_gin extension

### CODE REVIEW

**Review by truboxl — CHANGES_REQUESTED** [review]
Change your whitespaces to tabs and bump TERMUX_PKG_REVISION

**Comment by truboxl** [comment]
Please fix your tabbing

**Comment by TomJo2000** [comment]
All Shell scripts (`.sh` files) should be indented using tabs, as per the coding guidelines.
(Apparently we do not specify a tab width for them, but 4 is customary for `build.sh` files)
https://github.com/termux/termux-packages/wiki/Coding-guideline#script-formatting

We also provide a [`.editorconfig` file in the repo](https://github.com/termux/termux-packages/blob/master/.editorconfig), to automatically apply these guidelines,
please make sure your editor is properly using it.
For additional information see: https://editorconfig.org/

**Comment by TomJo2000** [comment]
That's just a web version of VS Code, so that does support Editorconfig, with a plugin.

File: packages/postgresql/build.sh [hunk 1]
```diff
@@ -57,6 +58,7 @@ termux_step_post_make_install() {
 	make -C doc/src/sgml install-man
 
 	for contrib in \
+                btree_gin \
```
Comment (truboxl) [inline packages/postgresql/build.sh:61]:
```suggestion
		btree_gin \
```

**Comment by TomJo2000** [comment]
It seems to be building, but the commits are a mess.
We ***can*** squash that together during merge,
but we usually prefer the PR author to keep their commit(s) organized.

You have stated that you are using GitHub Web on a mobile phone to make this PR, so I'm inclined to be more lenient on commit cleanliness, I am still going to complain about it though.

---

More importantly, testing would be appreciated for the PR.

<sup>(This is a pre-written, saved reply.)</sup>
If you want to test this PR please download the appropriate DEB package(s)
from the build artifacts of the [associated PR's latest CI run](https://github.com/termux/termux-packages/pull/19780/checks).
image
As far as I am aware build artifact downloads are not available through the GitHub mobile app,
so you will need to open this PR in the browser to download them.

After downloading the build artifact, make sure to `unzip` and un-`tar` it.
<details><summary>Detailed instructions, if needed.</summary>
<p>

```bash
# finding out what architecture you need
# architecture is just below the TERMUX_VERSION
termux-info

# e.g.
# [...]
# TERMUX_MAIN_PACKAGE_FORMAT=debian
# TERMUX_VERSION=0.118.0
# TERMUX__USER_ID=0
# Packages CPU architecture:
# aarch64
# [...]

# =======================

# make sure `unzip` and `tar` are installed using
pkg install unzip tar

# unzip the artifact (if you have a different architecture this might be arm, i686 or x86_64 instead)
unzip debs-aarch64-*.zip

# untar the artifact
tar xf debs-aarch64-*.tar

# You should now have a debs/ directory in your current working directory
# Install the packages from the local source using
pkg install -- ./debs/*.deb

# to clean up, you can remove the debs/ directory, .tar file and .zip file
rm -rfi debs debs-aarch64-*.zip debs-aarch64-*.tar
```

</p>
</details>

**Review by truboxl — APPROVED** [review]
Nvm. I will squash it.

**Pull request merged**